test/AxisTests.js

```diff
@@ -19,19 +19,22 @@ import {
 import viewconf from './view-configs/axis';
 
 configure({ adapter: new Adapter() });
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 30000;
```

Comment:
Is this something that should be set per test or globally?

Reply:
Good idea. I'll set it globally.